stock_grn/security/ir_rule.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+  <record model="ir.rule" id="grn_rule">
+    <field name="name">grn company rule</field>
+    <field name="model_id" ref="model_stock_grn" />
+    <field name="global" eval="True" />
+    <field
+            name="domain_force"
+        >['|',('company_id','=',False),('company_id','=',user.company_id.id)]</field>
```

Comment:
```suggestion
        >['|',('company_id','=',False),('company_id','in',company_ids)]</field>
```

Reply:
@glitchov I suggested 'in' :sweat_smile:

Reply:
Corrected in between ;-)